src/quo/components/list_items/network_list/view.cljs

```diff
@@ -0,0 +1,76 @@
+(ns quo.components.list-items.network-list.view
+  (:require
+    [clojure.string :as string]
+    [quo.components.list-items.network-list.style :as style]
+    [quo.components.markdown.text :as text]
+    [quo.foundations.resources :as quo.resources]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [schema.core :as schema]))
+
+(defn- info
+  [{:keys [network-name label]}]
+  [rn/view {:style style/info}
+   [rn/image
+    {:source (quo.resources/get-network network-name)
+     :style  style/network-image}]
+   [rn/view
+    [text/text
+     {:weight          :semi-bold
+      :style           {:text-transform :capitalize}
+      :number-of-lines 1}
+     (if (string/blank? label) "-" label)]]])
+
+(defn- values
+  [{:keys [token-value fiat-value theme]}]
+  [rn/view {:style style/values-container}
+   [text/text
+    {:weight          :medium
+     :size            :paragraph-2
+     :number-of-lines 1}
+    token-value]
+   [text/text
+    {:style           (style/fiat-value theme)
+     :size            :paragraph-2
+     :number-of-lines 1}
+    fiat-value]])
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:network-name {:optional true} [:maybe keyword?]]
+      [:fiat-value {:optional true} [:maybe :string]]
+      [:label {:optional true} [:maybe :string]]
+      [:token-value {:optional true} [:maybe [:string]]]
+      [:customization-color {:optional true} [:maybe :schema.common/customization-color]]
+      [:state {:optional true :default :default} [:maybe keyword?]]
```

Comment:
What's the purpose of `:default :default`? I can see that `:state` can be either `:pressed :transparent :active`, so it's better to use 
```clj
[:state {:optional true} [:maybe [:enum :pressed :transparent :active]]]
```

Reply:
updated 👍

Reply:
sorry I added an `EDIT` to my previous message earlier. `:transparent` is the default value for the background color, but it isn't the default state. It should be 
```clj
[:state {:optional true} [:maybe [:enum :pressed :active]]]
```